dcpy/models/product/dataset/metadata.py

```diff
@@ -40,23 +43,18 @@ class DatasetOverrides(BaseModel, extra="forbid"):
     display_name: str | None = None
     description: str | None = None
     tags: list[str] = []
-    destination_file_name: str | None = None
+    destination_file_name: str | None = None  # TODO filename
 
-    _VERSION_TEMPLATE_TOKEN = "{{ version }}"
 
-    def get_dataset_destination_name(self, version) -> str:
-        """Get filename for the destination dataset. Should only be invoked for unparsed datasets."""
-        return (
-            self.destination_file_name.replace(self._VERSION_TEMPLATE_TOKEN, version)
-            if self.destination_file_name is not None
-            else ""
-        )
+class RemoteFile(BaseModel, extra="forbid"):
+    id: str
```

Comment:
introducing the concept of a `file.id`, to refer to a file in the package

Reply:
Yeah this makes a lot of sense if we're having the same file with different filenames in different places